### ISSUE TITLE
Need to determine how to assign new Block Types/Option Codes in future

### ISSUE THREAD

**Issue by hadrielk** [pcapng/pcapng#23]
Right now the doc says to contact the authors to assign new block types. That's not really sensible, imo.

I think one reasonable approach is to ask IANA for new registries for them, and pick a policy like "Designated Expert" or even "Specification Required". Because for experimental or proprietary uses, we have Custom Blocks and Custom Options.

IANA creates registries all the time, and they work reasonably well.

Also, we need to include NRB Record Types for that model as well. (I don't see any others, but there might be more)

**Comment by packetfoo** [comment]
Yep, I agree. The last time someone needed new block types was when Loris asked for sysdig, and there wasn't a clear process.
